changelogs/unreleased/6211-support-not-in,.yml

```diff
@@ -1,6 +1,6 @@
-description: Add the syntactic sugar "value not in items" for "not (value in items)"
+description: Add the "not in" operator
```

Comment:
This is possibly invalid yaml, you may have to quote the entire thing in single quotes. I'm not sure. The merge tool will tell you if it's invalid.